include/alpaka/atomic/AtomicOmpBuildIn.hpp

```diff
@@ -0,0 +1,326 @@
+/**
+* \file
+* Copyright 2014-2018 Benjamin Worpitz, Rene Widera
+*
+* This file is part of alpaka.
+*
+* alpaka is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* alpaka is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with alpaka.
+* If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#ifdef _OPENMP
+
+#include <alpaka/atomic/Traits.hpp>
+
+namespace alpaka
+{
+    namespace atomic
+    {
+        //#############################################################################
+        //! The OpenMP accelerator atomic ops.
+        //
+        //  Atomics can be used in the blocks and threads hierarchy levels.
+        //  Atomics are not guaranteed to be save between devices or grids.
+        class AtomicOmpBuildIn
```

Comment:
Could you please fix the spelling and use `AtomicOmpBuiltIn`?

Reply:
thx, I fixed it and forced push the changes